### DISCUSSION TITLE
LoRAs for my character

### DISCUSSION THREAD

**Discussion by Jano14** [lllyasviel/Fooocus#2444] [Q&A] [answered]
Hi Again!!

I want to generate LoRAs to have a consistent character. What do you recommend, create two LoRAs, one for the face and one for the positions, or create a single LoRA that includes everything?

By creating two LoRAs, there will be no conflicts between the face of the first LoRA and the faces of the poses LoRA? Could you guide me a little on that? The objective is to enter my name and for the prompt to know what face to give it and then the positions in a more standard way. Do you know of any recommended LoRA for the poses? One standard, well known and already tested.

Thanks in advance!!

**Comment by mashb1t** [comment, marked as answer]
i'd recommend one LoRA for the face/person and then the use of ControlNet OpenPose for the pose, if it is important to you.
Fooocus does not (yet) support OpenPose, but there are users planning on migrating the preprocessor and processing from https://github.com/fenneishi/Fooocus-ControlNet-SDXL to this repo, see https://github.com/lllyasviel/Fooocus/discussions/1324#discussioncomment-8661590.
